scripts/jest/jest-cli.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const {spawn} = require('child_process');
+const chalk = require('chalk');
+const yargs = require('yargs');
+
+const configMap = {
+  source: './scripts/jest/config.source.js',
+  www: './scripts/jest/config.source-www.js',
+  persist: './scripts/jest/config.source-persistent.js',
+  build: './scripts/jest/config.build.js',
+  devtools: './scripts/jest/config.build-devtools.js',
+};
+
+const argv = yargs
+  .parserConfiguration({
+    // Important: This option tells yargs to move all other options not
+    // specified here into the `_` key. We use this to send all of the
+    // Jest options that we don't use through to Jest (like --watch).
+    'unknown-options-as-args': true,
+  })
+  .options({
+    debug: {
+      alias: 'd',
+      describe: 'Run with node debugger attached.',
+      requiresArg: false,
+      type: 'boolean',
+      default: false,
+    },
+    prod: {
+      alias: 'p',
+      describe: 'Run with NODE_ENV=production.',
+      requiresArg: false,
+      type: 'boolean',
+      default: false,
+    },
+    variant: {
+      alias: 'v',
+      describe: 'Run with www variant set to true.',
+      requiresArg: false,
+      type: 'boolean',
+      default: false,
+    },
+    config: {
```

Comment:
What's the use case for this? I'd prefer to think of the config files as an implementation detail. If there are config files aren't covered by the other options, might as well create an ad hoc test command.

Reply:
Short term my goal was to make it easy to match the existing test scripts which are basically narrowed to these configs as:

- test
- test-www
- test-persistent
- test-build
- test-build-devtools

I'd agree that the configs are implementation details, but I also don't have a lot of context on what the other options should be and how they relate to which config should be run.

Reply:
Looks like there are three "configs" (we need a better name, maybe "projects"):

- oss
- www
- devtools